<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewTransaction</class>
 <widget class="QWidget" name="NewTransaction">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>260</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>311</width>
    <height>223</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>101</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>From Account:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="FromAccountBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>191</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>To Account:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ToAccountBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>191</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>361</width>
     <height>101</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="ammountLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>16</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ã–sszeg</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="dateLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>191</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>nem ismert</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ammountEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="rentButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>230</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
